ReactWindows/ReactNative.Shared/UIManager/UIImplementation.cs

```diff
@@ -21,7 +21,7 @@ namespace ReactNative.UIManager
     /// <remarks>
     /// TODO: Animation support
     /// </remarks>
-    public class UIImplementation
+    public class UIImplementation : IEventEmitter
```

Comment:
We use property injection to set the EventDispatcher for this component... The preferable approach would be to use constructor injection, however, the UIIMplementation is constructed outside the UIManagerModule and passed into the UIManagerModule via its constructor... While the EventDispatcher is created inside the UIManagerModule constructor...